cmd/clean.go

```diff
@@ -22,16 +23,21 @@ func init() {
 
 // CleanCommand cleans up files
 func CleanCommand(cmd *cobra.Command, args []string) error {
+	initConfig()
+
 	filesToClean, err := fileutil.SomeOrAllFiles(args, sopsConfig.EncryptedFiles)
 	if err != nil {
 		return err
 	}
 
 	//clean all the files
 	for _, f := range filesToClean {
-		err := execwrap.RemoveFile(f)
+		_, err := os.Stat(f)
 		if err != nil {
-			return err
+			err = os.Remove(f)
+			if err != nil {
+				return err
+			}
```

Comment:
This makes more sense to me now. Better to only bail if it can't remove a file that's there.